src/addrdb.cpp

```diff
@@ -199,11 +199,13 @@ bool CBanDB::Write(const banmap_t& banSet)
 
 bool CBanDB::Read(banmap_t& banSet, bool& dirty)
 {
-    // If the JSON banlist does not exist, then try to read the non-upgraded banlist.dat.
+    if (fs::exists(m_banlist_dat)) {
+        LogPrintf("banlist.dat ignored because it can only be read by " PACKAGE_NAME " version 22.x. Remove %s to silence this warning.", m_banlist_dat);
```

Comment:
`All calls to LogPrintf() and LogPrint() should be terminated with \n`

Reply:
Thanks, fixed.